src/fluree/db/query/fql/syntax.cljc

```diff
@@ -50,6 +52,10 @@
   (when (map? x)
     (-> x first key)))
 
+(defn string->keyword
+  [x]
+  (cond-> x (string? x) keyword))
```

Comment:
Might `(if (string? x) (keyword x) x)` be a little clearer here? Might be good to mention that it returns non-strings unchanged in its docstring too.